File: Day-1/prog-2.c
#include<stdio.h>
int main()
{
    printf("enter a number= ");
    int n;
    scanf("%d", &n);
    int pro;
    for(int i=1; i<=10; i++)
    {
        pro= n*i;
        printf("%d * %d = %d \n", n,i, pro);
    }
    return 0;
}

File: Day-5/Prog2.java
import java.util.Scanner;
public class Prog2 {
     static int fact(int n)
    {
        int f=1;
        for(int i=1; i<=n; i++)
        {
            f*=i;
        }
        return f;
    }
    public static void main(String args[])
    {
        Scanner in= new Scanner(System.in);
        System.out.println("enter a number=");
        int n= in.nextInt();
        int num=n, sum=0;
        while(n!=0)
        {
            int d= n%10;
            sum+= fact(d);
            n/=10;
        }
        if(sum==num)
            System.out.println("the number is strong.");
        else
            System.out.println("number is not strong.");
        in.close();


    }
}
